respy/tests/random_model.py

```diff
@@ -269,7 +272,7 @@ def simulate_truncated_data(params, options, is_missings=True):
 
     # We can restrict the information to observed entities only.
     labels, _ = generate_column_labels_estimation(optim_paras)
```

Comment:
Can we just use generate col labels simulation here instead of subsetting two lines below?

Reply:
I do not think so because the simulation labels comprise a lot more variables. But, you are right that the function should have a different name. Something related to the state space.

Reply:
That makes sense!